server/cluster/unsafe_recovery_controller_test.go

```diff
@@ -669,7 +669,11 @@ func (s *testUnsafeRecoverSuite) TestSplitPaused(c *C) {
 		c.Assert(cluster.PutStore(store.GetMeta()), IsNil)
 	}
 	recoveryController := newUnsafeRecoveryController(cluster)
-	cluster.unsafeRecoveryController = recoveryController
+	{
```

Comment:
This kind of Rust-style block is not common in Go, I prefer not to write like this 😂.

Reply:
Done